nixos/modules/services/torrent/flood.nix

```diff
@@ -0,0 +1,287 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.flood;
+in
+{
+  options = {
+    services = {
+      flood = {
+        enable = lib.mkEnableOption (lib.mdDoc "Flood daemon");
```

Comment:
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557